addons/calendar/tests/test_calendar.py

```diff
@@ -381,6 +381,30 @@ def _test_emails_has_attachment(self, partners):
         # no more email should be sent
         _test_one_mail_per_attendee(self, partners)
 
+        # Check for attachments when public user creates an event ( simulates scheduling appointment from an incognito window)
+        attachment_2 = self.env['ir.attachment'].create({
```

Comment:
Can't we directly add the second attachment at beginning of test and consider we do all tests in this function with 2 attachments ?

Reply:
yup, moved it in there